swri_transform_util/nodes/initialize_origin.py

```diff
@@ -90,25 +91,29 @@ def __init__(self):
                                                   local_xy_navsatfix_topic,
                                                   self.navsat_callback, 2)
             self.subscribers = [gps_sub, navsat_sub]
-        else:
-            try:
-                self.get_logger().info("local_xy_origins_param[0]: %lf" % self.local_xy_origins_param.value[0]) 
-                self.get_logger().info("local_xy_origins_param[1]: %lf" % self.local_xy_origins_param.value[1]) 
-                origin_dict={'latitude': self.local_xy_origins_param.value[0],  
-                             'longitude':self.local_xy_origins_param.value[1],  
-                             'altitude':self.local_xy_origins_param.value[2],   
-                             'heading':self.local_xy_origins_param.value[3]}
-            except rclpy.exceptions.ParameterException:
-                message = 'local_xy_origin is "{}", but local_xy_origins is not specified'
-                self.get_logger().fatal(message.format(self.local_xy_origin_param.value))
-                exit(1)
+        elif type(self.local_xy_origins_param.value) == list:
+            if (len(self.local_xy_origins_param.value) != 3):
+               self.get_logger().fatal(f'{self.local_xy_origins_param.name} should have len 3 [lat, lon, alt]')
```

Comment:
We have typically used the format `[lat, lon, alt, heading]` in case you needed some rotational offset from ENU. I've never personally used it, but is there a good reason to remove it?

Reply:
initialize_origin and origin_manager have (as far as I know) never used the heading field. So including it would technically give as a smooth path to adding it, but kinda lie about it existing in the mean time (it sure fooled me).

The LocalXyUtil in the transform_manager can use it, but it hasn't been passed along all this time.

Reply:
Yeah, I guess I'm in favor of keeping it so we can go back and actually pass down that information and get that functionality

Reply:
I agree with @rjb0026 .

Reply:
This and the relevant tests have been reverted to the length 4 array of `[lat, lon, alt, heading]`

Reply:
See #755